docs/src/main/asciidoc/bigquery.adoc

```diff
@@ -0,0 +1,38 @@
+== Google Cloud BigQuery
+
+https://cloud.google.com/bigquery[Google Cloud BigQuery] is a fully managed, petabyte scale, low cost analytics data warehouse.
+
+Spring Cloud GCP provides:
+
+* A convenience starter which automatically configures the https://googleapis.dev/java/google-cloud-clients/latest/com/google/cloud/bigquery/BigQuery.html[`BigQuery`] client object with credentials needed to interface with BigQuery.
+
+Maven coordinates, using <<getting-started.adoc#_bill_of_materials, Spring Cloud GCP BOM>>:
+
+[source,xml]
+----
+<dependency>
+    <groupId>org.springframework.cloud</groupId>
+    <artifactId>spring-cloud-gcp-bigquery-starter</artifactId>
+</dependency>
+<dependency>
+    <groupId>org.springframework.integration</groupId>
+    <artifactId>spring-integration-core</artifactId>
+</dependency>
+----
+
+Gradle coordinates:
+
+[source,subs="normal"]
+----
+dependencies {
+    compile group: 'org.springframework.cloud', name: 'spring-cloud-gcp-bigquery-starter'
+    compile group: 'org.springframework.integration', name: 'spring-integration-core'
+}
+----
+
+=== BigQuery Autoconfiguration
+
+The `GcpBigQueryAutoConfiguration` class configures an instance of https://googleapis.dev/java/google-cloud-clients/latest/com/google/cloud/bigquery/BigQuery.html[`BigQuery`] for you by inferring your credentials and Project ID from the machine's environment.
```

Comment:
Document `spring.cloud.gcp.bigquery.*` properties for consistency with other docs for starters?